pkg/cfn/manager/template.go

```diff
@@ -16,5 +19,15 @@ func (c *StackCollection) GetStackTemplate(stackName string) (string, error) {
 		return "", err
 	}
 
-	return *output.TemplateBody, nil
+	return ensureJSONResponse([]byte(*output.TemplateBody))
+}
+
+func ensureJSONResponse(templateBody []byte) (string, error) {
+	//since json is valid yaml we just need to check the response is valid yaml
+	template, err := goformation.ParseYAML(templateBody)
```

Comment:
I don't completely trust goformation and this function not to do weird things but it seems fine

Reply:
Yeah I was quite wary of some the special syntax getting lost, so I made the test a bit more robust to ensure things like the functions/ !Refs do get preserved